files/galaxy/tpv/roles.yml

```
@@ -6,3 +6,16 @@ roles:
   storage-test*:
     params:
       object_store_id: "s3_netapp01"
+
+  rstudio-poweruser*:
+    rules:
+      - id: rstudio_poweruser
+        if: |
+          'rstudio' in tool.id
```

```suggestion
          'rstudio' in tool.id and 'interactive' in tool.id
```

Maybe a bit stricter?